### DISCUSSION TITLE
idea-tips/debug/debug-backward-jump-to-line

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#17] [Announcements]
# idea-tips/debug/debug-backward-jump-to-line

上一篇关于[调试操作回退的介绍]中，我们采用了Reset Frame（Drop Frame）来实现。但该操作有一定的局限性，所以，这篇我们将介绍一个插件来帮助弥补Reset Frame做不到的一些回退操作。

https://www.didispace.com/idea-tips/debug/debug-backward-jump-to-line.html

**Comment by huxiaoning** [comment]
这个插件还有一个妙用，就是如果你有一个if else分支结构，当你断点调试时，代码执行路径只走其中一个分支，你无法覆盖到另一个分支的时候，可以用这个插件把执行位置往前移动，然后在这个分支判断处，强行把执行位置拖入那个你想执行但是一直执行不到的分支位置。
